### ISSUE TITLE
Modal Overrides

### ISSUE THREAD

**Issue by kyleharper** [gathercontent/gather-ui#42]
After some testing I noticed that if we removed a large amount of the modal styling, it makes no difference to the components styling.

For example;

- add `modal--large` to a modal component
- remove the following styles from `lib/Modal/styles.scss`;

```
left: 0;
right: 0;
width: auto;
margin: auto;
background-color: transparent;
box-shadow: none;
border: none;
```

- testing in `gather-content-ui` removing these don't change the outlook of the component
- this is the same for a lot of the styling for the Modal component

Further investigation points that we're overriding the styles of bootstrap with the exact same values which is confusing. If we want to remove Bootstrap from our deps then lets do so but at the moment its very difficult to see what Bootstrap is meant to handle and what our styles are. Especially for the Modal component.

I recommend we clean out everything that isn't required.

**Comment by kyleharper** [comment]
This may also be the case for most of `overrides/_modal.scss` which was abstracted from the modal component stylesheet in #41 .

**Comment by kyleharper** [comment]
Fixed in #106
